src/kata7.getEmployerRole.js

```diff
@@ -1,3 +1,6 @@
-const getEmployerRole = (employeeName, employees) => {};
+const getEmployerRole = (employeeName, employees) => {
+  const result = employees.find((item) => item.name === employeeName);
+  return result.role;
```

Comment:
This can also be returned in one line.
You could just add the .role to the end of the employees.find() method, delete the {} and the return:
`````
const getEmployerRole = (employeeName, employees) =>
  employees.find((item) => item.name === employeeName).role;

`````